mbtiles/src/mbtiles.rs

```diff
@@ -849,4 +1028,115 @@ mod tests {
         assert!(matches!(result, Err(MbtError::AggHashMismatch(..))));
         Ok(())
     }
+
+    #[actix_rt::test]
+    async fn meter_to_lnglat() {
+        let xy1 = (-20037508.34, -20037508.34);
+        let lonlat1 = (-179.99999997494382, -85.05112877764508);
+
+        let xy2 = (20037508.34, 20037508.34);
+        let lonlat2 = (179.99999997494382, 85.05112877764508);
+
+        let xy3 = (0.0, 0.0);
+        let lonlat3 = (0.0, 0.0);
+
+        let xy4 = (3000.0, 9000.0);
+        let lonlat4 = (0.026949458523585643, 0.08084834874097371);
+
+        assert_eq!(webmercator_to_wgs84(xy1), lonlat1);
```

Comment:
not a good practice to separate input values with their expected values from the actual `assert` calls.  Inline all variables. 

Also, it is futile to compare floating point numbers exactly. I suggest you add [approx](https://docs.rs/approx/latest/approx/index.html) as a `dev-dependency`, and use it to do the comparison with the epsilon parameter.

Reply:
Approx added as a dev-dependency now